x-pack/test/upgrade/services.ts

```diff
@@ -7,8 +7,10 @@
 
 import { services as functionalServices } from '../functional/services';
 import { services as reportingServices } from './reporting_services';
+import { services as mapsServices } from './maps_services';
```

Comment:
Can you rename mapsServices to something like mapSmokescreenServices or mapsUpgradeServices?